defaults/defaults_snapshotter_linux.go

```diff
@@ -14,7 +14,7 @@
    limitations under the License.
 */
 
-package containerd
+package defaults
 
 const (
 	// DefaultSnapshotter will set the default snapshotter for the platform.
```

Comment:
I think I commented  on another thread but might want to consider renaming the const to `Snapshotter`, so that it's used as `defaults.Snapshotter`

Reply:
@thaJeztah it looks to me like all other constants in the `defaults` package begin with `Default` (ex. `DefaultRootDir`, `DefaultAddress`, etc.). I wanted to double check that breaking the naming consistency with the other constants is acceptable before I make that change.